server/task_executor_schema_update_ghost_sync.go

```diff
@@ -177,23 +176,22 @@ func executeSync(ctx context.Context, l *zap.Logger, task *api.Task, mi *db.Migr
 		return -1, "", err
 	}
 
-	insertedID, err := util.BeginMigration(ctx, executor, mi, prevSchemaBuf.String(), statement, databaseName)
+	insertedID, err := util.BeginMigration(ctx, executor, mi, prevSchemaBuf.String(), statement, db.BytebaseDatabase)
 	if err != nil {
 		return -1, "", err
 	}
 	startedNs := time.Now().UnixNano()
 
 	defer func() {
-		if err := util.EndMigration(ctx, l, executor, startedNs, insertedID, updatedSchema, databaseName, resErr == nil /*isDone*/); err != nil {
+		if err := util.EndMigration(ctx, l, executor, startedNs, insertedID, updatedSchema, db.BytebaseDatabase, resErr == nil /*isDone*/); err != nil {
```

Comment:
Add a comment on why we use bytebase database instead of databaseName?

Reply:
`databaseName := db.BytebaseDatabase`
https://github.com/bytebase/bytebase/pull/1318/files/dfd6f61feb4c8f26785aff8c8e5b850fa9f3beab#diff-3e0a9de3a7e46533a48e3e6475f6ec330a2ac85dafc377ae7b19c8bc06a7da2bL158

I just want to make it clear that we are using a constant instead of a variable.

Reply:
Make sense.